ci-checks.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env nix-shell
 #!nix-shell -i bash
 
+set -eux
```

Comment:
:facepalm: thanks!

Reply:
No worries, I didn't catch it before in review =).